.travis.yml

```diff
@@ -40,10 +42,21 @@ env:
   - NPM_TAG=latest IMAGE_NAME=theia-php NODE_VERSION=10
   - NPM_TAG=next IMAGE_NAME=theia-php NODE_VERSION=10
 
+jobs:
+  # add installer builds to jobs matrix
+  include:
+    - &publish
+      env:
+      install: skip
+      script: cd theia-electron && yarn && yarn package
+      deploy: skip
+    - <<: *publish
+      os: osx
```

Comment:
@thegecko It looks unlikely we'll want to publish from here for Windows/Mac, since the signing will not work here. I am happy to keep these entries here for now and remove them when we have this working elsewhere. Sounds like a plan?